blaze/compute/tests/test_pandas_compute.py

```diff
@@ -709,6 +709,106 @@ def test_str_lower():
     result = compute(expr, df).reset_index(drop=True)
     assert_series_equal(expected, result)
 
+@pytest.mark.parametrize('old,new,max,expected', (
+    ('o', 'x', None, 'Sxmexne'),
+    ('o', 'x', 1, 'Sxmeone')))
+def test_str_replace(old, new, max, expected):
+    df = pd.Series(['Someone'], name='name')
```

Comment:
I have done that, and the tests still pass, without any changes to the functions' `schema` member.